dist/index.js

```diff
@@ -12954,108 +12966,137 @@ const details = (action, resources, operator) => {
-          .map(comment => {
-            return minimizeComment({ id: comment.id })
-              .catch(error => core.error(`Failed to minimize comment ${comment.id}: ${error.message}`));
-          });
-
-        return Promise.all(minimizePromises)
-          .then(() => core.info('All minimize operations completed.'))
-          .catch(error => core.error(`Error during minimize operations: ${error.message}`));
-      })
-      .catch(error => core.error(`Failed to retrieve comments: ${error.message}`));
+    })
+    .catch((error) =>
+      core.error(`Failed to retrieve comments: ${error.message}`),
+    );
 };
 
-
 try {
-    let rawOutput = output();
-    let createComment = true;
-
-    console.log("hidePreviousComments", hidePreviousComments)
-    console.log("hidePreviousComments && context.eventName === pull_request", hidePreviousComments && context.eventName === 'pull_request')
-    if (hidePreviousComments && context.eventName === 'pull_request') {
-      hideComments();
-    }
-
-    console.log("includePlanSummary", includePlanSummary)
-    if (includePlanSummary) {
-      core.info("Adding plan output to job summary")
-      core.summary.addHeading('Terraform Plan Results').addRaw(rawOutput).write()
-    }
-
-    console.log("quietMode", quietMode)
-    console.log("hasNoChanges", hasNoChanges)
-    console.log("quietMode && hasNoChanges", quietMode && hasNoChanges)
-    if (quietMode && hasNoChanges) {
-      core.info("quiet mode is enabled and there are no changes to the infrastructure.")
-      core.info("Skipping comment creation.")
-      createComment = false
-    }
-
-    if (context.eventName === 'pull_request') {
-      core.info(`Found PR # ${context.issue.number} from workflow context - proceeding to comment.`)
-    } else {
-      core.info("Action doesn't seem to be running in a PR workflow context.")
-      core.info("Skipping comment creation.")
-      createComment = false
-    }
-
-    if (createComment) {
-      core.info("Adding comment to PR");
-      core.info(`Comment: ${rawOutput}`);
-      octokit.rest.issues.createComment({
-        issue_number: context.issue.number,
-        owner: context.repo.owner,
-        repo: context.repo.repo,
-        body: rawOutput
-      });
-      core.info("Comment added successfully.");
-    }
+  let rawOutput = output();
+  let createComment = true;
+
+  console.log("hidePreviousComments", hidePreviousComments);
+  console.log(
+    "hidePreviousComments && context.eventName === pull_request",
+    hidePreviousComments && context.eventName === "pull_request",
+  );
+  if (hidePreviousComments && context.eventName === "pull_request") {
+    hideComments();
+  }
+
+  console.log("includePlanSummary", includePlanSummary);
+  if (includePlanSummary) {
+    core.info("Adding plan output to job summary");
+    core.summary.addHeading("Terraform Plan Results").addRaw(rawOutput).write();
+  }
+
+  console.log("quietMode", quietMode);
```

Comment:
Forgive me, do we want these console.logs? Or were these for debugging purposes?

Reply:
They look like debugging for sure. I did not add these and I think in general you should use the github `core.<info|debug>` library calls for debugging anyways. I will say I did not add and when I saved in my IDE it decided to add `;` and switch `'` to `"` so now I have touched everything